### DISCUSSION TITLE
Reorder custom block items

### DISCUSSION THREAD

**Discussion by FlamedDogo99** [ScratchAddons/ScratchAddons#6373] [Ideas]
When you are editing a custom block and you wish to add an item, such as a Boolean, reporter, or text, you can only add it to the end of the block. If you wish to add something at the beginning, you must delete each item, add the new item, and add back in the deleted items. This is problematic because it requires the user to update the replaced items in their project, due to their id changing. Because of this, I propose implementing a feature that allows the user to re order the existing items of a custom block in the edit screen. This is not the most important, but I feel that it is a small quality of life change that would improve the overall experience.

**Comment by WorldLanguages** [comment]
See #6073

**Reply by FlamedDogo99** [reply]
Thanks, sorry I missed that.
